dbux-code/src/terminal/TerminalWrapper.js

```diff
@@ -84,7 +84,7 @@ export default class TerminalWrapper {
     // const runJsCommand = `pwd && node -v && which node && echo %PATH% && node ${pathToDbuxRun} ${serializedRunJsArgs}`;
     const runJsCommand = `"${pathToNode}" "${pathToDbuxRun}" ${!!isInteractive + 0} ${serializedRunJsArgs}`;
 
-    debug('wrapping terminal command: ', JSON.stringify(runJsargs), `pathToDbuxRun: ${pathToDbuxRun}`);
+    Verbose && debug('wrapping terminal command: ', JSON.stringify(runJsargs), `pathToDbuxRun: ${pathToDbuxRun}`);
```

Comment:
Remove `Verbose && ` again.
This is not `Verbose`-only! We want to log certain crucial information - especially when it comes to things that are very platform-dependent, and thus can very easily go wrong. We want our log to show a lot of basic information.